Add translation files only if the NLS knob is set

--- src/sensors/CMakeLists.txt.orig	2019-12-05 06:33:57 UTC
+++ src/sensors/CMakeLists.txt
@@ -104,7 +104,9 @@ endforeach()
 
 # Translations
 
-mapper_translations_sources(
-  ${MAPPER_SENSORS_SOURCES}
-  ${POWERSHELL_POSITION_SOURCES}
-)
+if(Mapper_WITH_NLS)
+	mapper_translations_sources(
+	  ${MAPPER_SENSORS_SOURCES}
+	  ${POWERSHELL_POSITION_SOURCES}
+	)
+endif()
